src/diffusers/models/embeddings.py

```diff
@@ -371,6 +414,18 @@ def forward(self, text_embeds: torch.Tensor, image_embeds: torch.Tensor):
         embeds = torch.cat(
             [text_embeds, image_embeds], dim=1
         ).contiguous()  # [batch, seq_length + num_frames x height x width, channels]
+
+        if self.use_positional_embeddings:
```

Comment:
We now raise a cleaner error here with guidance on how to get around it. This will only ever happen for Cog-2B model btw since for 5B, we generate RoPE embeds in the pipeline and reuse at each denoising iteration.